### PR TITLE
Add truncate method

### PR DESCRIPTION
**Pull request by oleg1std**

### CODE REVIEW

File: HappyTravel.Money/Helpers/MoneyRound.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 namespace HappyTravel.Money.Helpers
 {
-    public static class MoneyCeiler
+    public static class MoneyRound
```
Comment (kirillta) [inline HappyTravel.Money/Helpers/MoneyRound.cs:8]: This is semantically wrong. It must be the _Round**er**_ or something like a _Processor_ which is better

File: HappyTravel.Money/Helpers/MoneyRound.cs [hunk 2]
```diff
@@ -18,6 +18,18 @@ public static decimal Ceil(decimal target, Currencies currency)
...
+        
+        public static decimal Truncate(decimal target, string currency) 
+            => Truncate(target, Enum.Parse<Currencies>(currency));
+        
+        
+        public static decimal Truncate(decimal target, Currencies currency) 
+            => Math.Round(target, CurrencyDecimalDigits[currency], MidpointRounding.ToZero);
+        
+        
         private static readonly Dictionary<Currencies, int> CurrencyDecimalDigits = new Dictionary<Currencies, int>
```
Comment (kirillta) [inline HappyTravel.Money/Helpers/MoneyRound.cs:33]: By the way could you move that to a safe public property?

**Comment by kirillta** [comment]
Why do we need to ground amounts anyway?

**Comment by oleg1std** [comment]
> Why do we need to ground amounts anyway?

The reason is that we have discounts.
For instance a discount amount can be 1400.235
If we round this value we don't want to increase the remaining number

**Pull request merged**